src/views/ModelPlan/CollaborationArea/Cards/ModelPlanCard/index.tsx

```diff
@@ -0,0 +1,251 @@
+  const [isExportModalOpen, setIsExportModalOpen] = useState<boolean>(false);
+  const { data, loading } = useGetModelPlanQuery({
+    variables: {
+      id: modelID
+    }
+  });
+
+  const modelPlan =
+    data?.modelPlan ||
+    ({
+      __typename: 'ModelPlan',
+      id: modelID,
+      modelName: 'Test',
+      archived: false,
+      status: ModelStatus.PLAN_DRAFT,
+      modifiedDts: '',
+      modifiedByUserAccount: {
+        __typename: 'UserAccount',
+        commonName: ''
+      },
+      basics: {
+        __typename: 'PlanBasics',
+        id: '123',
+        status: TaskStatus.READY
+      },
+      generalCharacteristics: {
+        __typename: 'PlanGeneralCharacteristics',
+        id: '54234',
+        createdBy: 'John Doe',
+        createdDts: '',
+        modifiedBy: '',
+        modifiedDts: '',
+        readyForClearanceDts: '',
+        status: TaskStatus.IN_PROGRESS
+      },
+      participantsAndProviders: {
+        __typename: 'PlanParticipantsAndProviders',
+        id: '46246356',
+        createdBy: 'John Doe',
+        createdDts: '',
+        modifiedBy: '',
+        modifiedDts: '',
+        readyForClearanceDts: '',
+        status: TaskStatus.IN_PROGRESS
+      },
+      beneficiaries: {
+        __typename: 'PlanBeneficiaries',
+        id: '09865643',
+        createdBy: 'John Doe',
+        createdDts: '',
+        modifiedBy: '',
+        modifiedDts: '',
+        readyForClearanceDts: '',
+        status: TaskStatus.IN_PROGRESS
+      },
+      opsEvalAndLearning: {
+        __typename: 'PlanOpsEvalAndLearning',
+        id: '7865676',
+        createdBy: 'John Doe',
+        createdDts: '',
+        modifiedBy: '',
+        modifiedDts: '',
+        readyForClearanceDts: '',
+        status: TaskStatus.IN_PROGRESS
+      },
+      payments: {
+        __typename: 'PlanPayments',
+        id: '8756435235',
+        createdBy: 'John Doe',
+        createdDts: '',
+        modifiedBy: '',
+        modifiedDts: '',
+        readyForClearanceDts: '',
+        status: TaskStatus.IN_PROGRESS
+      },
+      operationalNeeds: [],
+      taskListStatus: TaskStatus.READY,
+      isFavorite: true,
+      collaborators: [],
+      documents: [],
+      crs: [],
+      tdls: [],
+      discussions: [],
+      prepareForClearance: {
+        __typename: 'PrepareForClearance',
+        status: PrepareForClearanceStatus.IN_PROGRESS,
+        modifiedDts: ''
+      }
+    } as GetModelPlanTypes);
+
+  const {
+    modifiedDts,
+    modifiedByUserAccount,
+    basics,
+    generalCharacteristics,
+    participantsAndProviders,
+    beneficiaries,
+    opsEvalAndLearning,
+    payments,
+    operationalNeeds = [],
+    taskListStatus
+  } = modelPlan;
+
+  const getITSolutionsStatus = (
+    operationalNeedsArray: OperationalNeedsType[]
+  ) => {
+    const inProgress = operationalNeedsArray.find(need => need.modifiedDts);
+    return inProgress ? TaskStatus.IN_PROGRESS : TaskStatus.READY;
+  };
```

Comment:
I think this whole function can be moved outside the component.  It isn't dependent on outside state

Reply:
Also I don't think that is was initially addressed from the first comment.  We should be getting the modified via `opSolutionLastModifiedDts` on the model plan.